zhmcclient/_lpar.py

```diff
@@ -427,6 +427,150 @@ def deactivate(self, wait_for_completion=True,
+            Worldwide port name (WWPN) of the target SCSI device to be
+            used for this operation, in hexadecimal.
+
+          lun (:term:`string`):
+            Hexadecimal logical unit number (LUN) to be used for the
+            SCSI Load.
+
+          load_parameter (:term:`string`):
+            Optional load control string.  If empty string or `None`,
+            it is not passed to the HMC.
+
+          disk_partition_id (:term:`integer`):
+             Optional disk-partition-id (also called the boot program
+             selector) to be used for the SCSI Load.
+
+          operating_system_specific_load_parameters (:term:`string`):
+             Optional operating system specific load parameters to be
+             used for the SCSI Load.
+
+          boot_record_logical_block_address (:term:`string`):
+             Optional hexadecimal boot record logical block address to
+             be used for the SCSI Load.
+
+          wait_for_completion (bool):
+            Boolean controlling whether this method should wait for completion
+            of the requested asynchronous HMC operation, as follows:
+
+            * If `True`, this method will wait for completion of the
+              asynchronous job performing the operation, and for the status
+              becoming "operating" (or in addition "exceptions", if
+              `allow_status_exceptions` was set.
+
+            * If `False`, this method will return immediately once the HMC has
+              accepted the request to perform the operation.
+
+          operation_timeout (:term:`number`):
+            Timeout in seconds, for waiting for completion of the asynchronous
+            job performing the operation. The special value 0 means that no
+            timeout is set. `None` means that the default async operation
+            timeout of the session is used. If the timeout expires when
+            `wait_for_completion=True`, a
+            :exc:`~zhmcclient.OperationTimeout` is raised.
+
+          status_timeout (:term:`number`):
+            Timeout in seconds, for waiting that the status of the LPAR has
+            reached the desired status, after the HMC operation has completed.
+            The special value 0 means that no timeout is set. `None` means that
+            the default async operation timeout of the session is used.
+            If the timeout expires when `wait_for_completion=True`, a
+            :exc:`~zhmcclient.StatusTimeout` is raised.
+
+          allow_status_exceptions (bool):
+            Boolean controlling whether LPAR status "exceptions" is considered
+            an additional acceptable end status when `wait_for_completion` is
+            set.
+
+          force (bool):
+            Boolean controlling whether this operation is permitted when the
+            LPAR is in the "operating" status.
+
+            TBD: What will happen with the LPAR in that case (deactivated then
+            activated? nothing?)
```

Comment:
I suggest to move the "force" argument to before the "wait_for_completion" argument. It is one that is put into the request body, and those we usually have before those that the zhmcclient adds.